lib/pengine/common.c

```diff
@@ -85,6 +87,24 @@ static pe_cluster_option pe_opts[] = {
 	  "When set to TRUE, the cluster will immediately ban a resource from a node if it fails to start there. When FALSE, the cluster will instead check the resource's fail count against its migration-threshold." },
 	{ "enable-startup-probes", NULL, "boolean", NULL, "true", &check_boolean,
 	  "Should the cluster check for active resources during startup", NULL },
+    {
+        XML_CONFIG_ATTR_SHUTDOWN_LOCK,
+        NULL, "boolean", NULL, "false", &check_boolean,
+        "Whether to lock resources to a cleanly shut down node",
+        "When set to true, resources active on a node being cleanly shut down"
+            "are not allowed to start again until that node rejoins (or at "
+            "most shutdown-lock-expiration, if set). Clones, bundles, stonith "
```

Comment:
... or for at most ...

Reply:
Maybe we can make it somehow express the real purpose more clearly.
Atm nothing states that they are forced to the initial location - not even
stickiness in its original behavior as the resources are down.

Reply:
Documented